internal/cli/config/config.go

```diff
@@ -80,6 +88,13 @@ func (c *Config) SetFilePath(filePath string) {
 	c.filePath = filePath
 }
 
+func (c *Config) GetDacOutputFolder() string {
```

Comment:
if you want to keep a getter, then you should not expose the attribute `OutputFolder`.

Or you keep it exposed, then you remove this getter and instead you init this value in the method `init` above